src/views/components/search.html

```diff
@@ -129,7 +162,7 @@
 				});
 
 				// Update results on input.
-				this.observe('page query __ready', (newValue, oldValue, keypath) => {
+				this.observe('page query type __ready', (newValue, oldValue, keypath) => {
```

Comment:
You should be able to mix JS and CSS filters, so we should have separate `type` and `style` params.

Reply:
Yes, just split it into two separate parameters everywhere, including search. And the query will look like `bootstrap type:-none style:-none`

Reply:
I see. Thanks for your advice.